clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -360,8 +366,8 @@ public synchronized int createRawTable(TachyonURI path, int columns, ByteBuffer
       throw new IOException("Column count " + columns + " is smaller than 1 or " + "bigger than "
           + maxColumns);
     }
-
-    return mMasterClient.user_createRawTable(path.getPath(), columns, metadata);
+    throw new UnsupportedOperationException("Raw table is currently unsupported");
+    // return mMasterClient.user_createRawTable(path.getPath(), columns, metadata);
```

Comment:
Add a TODO to comment out this code when the support is re-enabled (or remove the dead code).